.github/workflows/rust.yml

```diff
@@ -40,20 +35,19 @@ jobs:
         rust: ['stable']
         include:
           - os: 'ubuntu-20.04'
-            rust: '1.41'
+            rust: '1.63'
     runs-on: ${{ matrix.os }}
     steps:
       - uses: actions/checkout@v2
```

Comment:
Looks like you need to bump this one too.

Reply:
Good catch, thanks!